tests/integration/util/test_file_converter.py

```diff
@@ -0,0 +1,65 @@
+"""Integration tests for file_converter.py"""
+import os
+import pytest
+
+from clean_air.util import file_converter as fc
+
+
+@pytest.fixture
+def xl_input_path(sampledir):
+    xl_data_path = os.path.join(sampledir, "test_data",
+                                "metadata_form_responses.xlsx")
+    return xl_data_path
+
+
+@pytest.fixture
+def tmp_output_path(tmp_path):
+    tmp_output_path = tmp_path / "test_data"
+    tmp_output_path.mkdir()
+    return tmp_output_path
+
+
+@pytest.fixture
+def json_filename(tmp_output_path):
+    json_fname = os.path.join(tmp_output_path, "form_response0.json")
+    return json_fname
+
+
+@pytest.fixture
+def yaml_filename(tmp_output_path):
+    yaml_fname = os.path.join(tmp_output_path, "form_response0.yaml")
+    return yaml_fname
+
+
+def test_convert_excel_to_json(xl_input_path, tmp_output_path,
+                               json_filename):
+    """
+    Test to check end-to-end processing of excel metadata form responses and
+    their conversion to reformatted json files.
+    """
+    # Run conversion and check for file in tmp_path:
+    fc.convert_excel_to_json(xl_input_path, tmp_output_path)
+    try:
+        with open(json_filename) as file:
+            file.read()
+    except FileNotFoundError as fnf_error:
+        print(fnf_error)
+        print("Unable to locate converted json file.")
```

Comment:
Yes, I think I was playing around with this for a while, wondering if it would be any help in the event of a test fail.  It didn't really help at all though; the test output was good enough without the print statement, so I will remove it.

Reply:
Hopefully this is ok now.

Reply:
Yes, there's just one other one to update too

Reply:
Could be, I'd hope the file not found error already includes the file name. But really I meant there were two tests that didn't re-raise the error and the latest commit only changed one of them.

Reply:
oh i do apologise, that's just me being stupid.  well i'll keep them as try/except clauses then, and just change thhhhhhe one that i missed.  let me know now if you want them simplified further.